package.json

```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.24.0",
```

Comment:
check the difference between axios and fetch and see what other functionality does axios provide